src/components/flat-table/flat-table-cell/flat-table-cell.component.tsx

```diff
@@ -59,8 +59,13 @@ export const FlatTableCell = ({
     isInHighlightedRow,
     isInSelectedRow,
     tabIndex,
+    bringToFront,
   } = useTableCell(internalId.current);
 
+  const handleOnFocus = useCallback((ev: React.FocusEvent<HTMLElement>) => {
```

Comment:
question: Is there any performance enhancement from `useCallback` with an empty array here? As we're not passing in the `bringToFront` as a dependency do we need to memoize at all or is the cost in memory overkill here?

https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
This has been refactored and the unnecessary `useCallback` has been removed in order to increase performance.